UNATTENDED.md

```diff
@@ -0,0 +1,74 @@
+
+## Language
+
+By default, the reformatter user interface will be shown in _English (US)_. To present the reformatter in a different language, add a section like this to `unattended.ini`:
+
+```ini
+[EndlessOS]
+locale=pt_BR.utf8
+```
+
+In the example above, the reformatter will be displayed in _Brazilian Portuguese_. You may list the available locales on an Endless OS system by running the following in a Terminal window:
+
+```console
+$ locale -a | grep utf8
+```
+
+Note that this setting has _no effect_ on the default language for the _installed_ system.
+
+## Target Computer
+
+To reduce the possibility of accidentally booting an unattended installer on a system with valuable data, we recommend that the disk is pre-configured with target computers. On these systems, installation will commence _without any keypresses required_, after a 30-second countdown showing what action will be taken.
+
+These are specified in `unattended.ini` with option groups starting with `Computer`. Within each entry, `vendor` and `product` must both be specified, otherwise the entry is ignored. Here is an example targeting two computers:
+
+```ini
+[Computer 1]
+vendor=Asus
+product=X441SA
+
+[Computer 2]
+vendor=GIGABYTE
+product=GB-BXBT-2807
+```
+
+The correct values for `vendor` and `product` for a given system can be found in `/sys/class/dmi/id/sys_vendor` and `/sys/class/dmi/id/product_name` respectively.
+
+If no computer entries are provided, all computers will be reformatted, but explicit confirmation will be required before the process begins. If computers are specified, using the installer on a different computer will show a warning, and again require explicit confirmation to proceed.
+
+## Image
+
+If there is only one Endless OS image file on the USB stick, and only one disk present in the system (ignoring the one the installer is running from), no configuration is needed: that image will be written to that disk. Otherwise, add an option group beginning `Image` specifying the `filename` and/or `blockdevice` to use:
+
+```ini
+[Image 1]
+filename=eos-eos3.3-amd64-amd64.180115-104625.en.img.gz
+blockdevice=sd
+```
+
+If specified, `blockdevice` can take one of two forms:
+
+* the base disk type (eg `sd` or `mmcblk`). In this case, eos-installer will list disks matching this name (eg `/dev/sda`, `/dev/sdb`), ignore any which correspond to the USB device the installer is running from, and ignore any where media is not present (eg SD card readers with no card inserted). If that leaves a single candidate, the installer can proceed unattended. Otherwise, the process fails.
```

Comment:
Nitpick: s/eg/e.g./ or, even better, stop using Latin expressions and just say ‘for example’. (As per §(Latin terms) on https://developer.gnome.org/gdp-style-guide/2.32/grammar.html.en.)